templates/workspace_services/guacamole/user_resources/guacamole-azure-linuxvm/terraform/vm_config.sh

```diff
@@ -42,44 +43,51 @@ if [ ${shared_storage_access} -eq 1 ]; then
   credentialRoot="/etc/smbcredentials"
 
   mntPath="$mntRoot/$fileShareName"
-  smbPath=$(echo $httpEndpoint | cut -c7-$(expr length $httpEndpoint))$fileShareName
+  # shellcheck disable=SC2308
+  smbPath=$(echo "$httpEndpoint" | cut -c7-"$(expr length "$httpEndpoint")")$fileShareName
```

Comment:
Fixing the warning here is a bit more of a change so I understand if you chose to ignore it. 2 options:
1. continue to use cut but replace expr
```bash
smbPath=$(echo "$httpEndpoint" | cut -c7-"${#httpEndpoint}")$fileShareName
```
2. do everything with just bash which is cleaner (I believe this remove the protocol prefix from the url)
```bash
smbPath="${httpEndpoint:7}${fileShareName}"
```

Please note that I haven't tested it.

Reply:
Yeah, not my code (most of this script I think was put in before linting was in the codebase) so was wary of changing too much - but given we've now fixed the other stuff I think we may as well address this also. I like the pure bash approach so will add this and re-test the linux VM

Reply:
Because of the template renderer it wont allow `:` in `${}` expressions so will go with option 1

Reply:
Neither of these options work unfortunately as the template renderer gives a `failed to render : <template_file>:49,17-18: Invalid character` when you have any expressions within handlebars `${}`. Since this isn't my code I'm not going to spend any more time looking into this and will disable the rule as I did originally